run.py

```diff
@@ -651,7 +659,12 @@ def speaker_info(speaker_uuid: str, core_version: Optional[str] = None):
                     for j in range(3)
                 ]
                 style_infos.append(
-                    {"id": id, "icon": icon, "voice_samples": voice_samples}
+                    {
+                        "id": id,
+                        "icon": icon,
+                        "style_portrait": style_portrait,
```

Comment:
```suggestion
                        "portrait": style_portrait,
```
`icon`に`style_`が付いていないので、これもつけなくてよさそう。

Reply:
個人的には親にも`portrait`があるので明確に分けた方がいいのではという気がします。
`style_infos`の子にあるということで十分かもしれませんが…
親要素にあるものを子が追加で持つという構造は他にはない気がする？

ここはAPIの部分なのでよく考えた方がいい気がします。

Reply:
あと`style_portrait`がないとき`None`を入れてしまうことに気づきました。

Reply:
ここは`StyleInfo`クラス内なので、`icon`にならって`portait`のがわかりやすいかなと思います！